cli/azd/pkg/commands/pipeline/pipeline.go

```diff
@@ -38,6 +40,10 @@ type gitRepositoryDetails struct {
 	repoName string
 	// System path where the git project is
 	gitProjectPath string
+	//Indicates if the repo was successfully pushed a remote
+	pushStatus bool
+
+	details interface{}
```

Comment:
Maybe add a comment here that says this is provider specific detail information or soemthing?  Also you might consider using `any` vs `interface{}` here (they are identical, any is just new in go 1.18, and reads a little nicer)

Reply:
https://github.com/Azure/azure-dev/issues/756